src/bin/server/app.rs

```diff
@@ -123,9 +123,9 @@ mod test {
         message
     }
 
-    #[tokio::test]
-    async fn can_aggregate_multiple_messages() {
-        let num_validators = 3;
+    #[sqlx::test]
```

Comment:
Oooooh, this works when running the GitHub CI postgres! Interesting. It'll likely fail with most cloud providers, it definitely does with Scaleway which means because the tests run against a cloud DB in eth-analysis I can't use this macro... annoying, maybe good reason to migrate to GitHub CI there 🤔 .

Reply:
Strange that it fails with scaleway, I'd expect it to be agnostic to the runtime environment.

Reply:
Haven't tried scaleway yet but among all the CI providers I tried (Jenkins, CircleCi, Github Actions) I prefer github just for the ease of use / lack of complexity and ecosystem of prebuilt actions.